src/main.rs

```diff
@@ -381,9 +415,32 @@ fn vmm_control_event(
     Ok(())
 }
 
+fn create_path(path: &str) -> &str {
+    let path_elems = path.split(':').collect::<Vec<&str>>();
+    match path_elems[0] {
+        "fifo" => {
+            if !Path::new(path_elems[1]).exists() {
+                let path_cstr = CString::new(path_elems[1])
+                    .expect("Could not create CString from the provided bytes.");
+                // Safe because we pass a null-terminated string to `mkfifo` and verify the result.
+                SyscallReturnCode(unsafe { libc::mkfifo(path_cstr.as_ptr(), 0o644) })
+                    .into_empty_result()
+                    .expect("Could not create fifo for the provided path.");
+            }
+        }
+        "file" => {
+            fs::File::create(PathBuf::from(path_elems[1]).as_path())
+                .expect("Could not create file for the provided path.");
+        }
+        _ => panic!("Wrong parameter provided for logger: has to be 'fifo' or 'file'."),
+    }
+    path_elems[1]
```

Comment:
What happens if someone pases `"file:"`?

Reply:
We panic with a `No such file or directory` message which I think it's okay.